index.html

```diff
@@ -3,17 +3,85 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <title>My Resume</title>
+  <title>The Party</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Eczar" rel="stylesheet">
+
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <!-- main wrapper -->
+  <div class="wrapper">
+    <!-- set partystyles later  -->
+    <nav>
+      <!-- nav block -->
+      <a class="navLink partyFlair" href="#">The Party</a>
+      <a class="navLink kenFlair" href="#">The Chronicler</a>
+      <a class="navLink bravaFlair" href="#">The Sorceress</a>
+      <a class="navLink oreFlair" href="#">The Gargoyle</a>
+    </nav>
+    <header>
+      <h1>Party Composition</h1>
+      <!-- Would like to put in less banter-y text maybe, but it wasn't as important as the styling. -->
```

Comment:
great job including comments not just on the code itself, but on your thoughts about the code and what changes you might want to do later.